less/about/siteDetails.less

```diff
@@ -82,6 +82,9 @@ body {
       font-size: 16px;
       margin-bottom: 12px;
       padding-left: @aboutPageSectionPadding;
+      width: 400px;
+      display: flex;
+      justify-content: space-between;
```

Comment:
I think maybe we should move these three styles to parent selector of `.sectionTitle`
so the original sectionTitle style will remain the same.
<img width="425" alt="screen shot 2016-12-09 at 13 25 49" src="https://cloud.githubusercontent.com/assets/11330831/21038759/3f8eaf5a-be13-11e6-87fa-9d3fb80c4aa6.png">